Cargo.toml

```diff
@@ -55,10 +55,24 @@ uuid = {version = "0.5", features = ["v4"]}
 [dev-dependencies]
 tempdir = "0.3"
 quickcheck = "0.6"
+criterion = "0.1"
+rand = "0.4"
 
 [profile.dev]
 codegen-units = 4
 
 [profile.release]
 lto = true
 codegen-units = 1
+
+[[bench]]
+name = "buckets"
+harness = false
+
+[[bench]]
+name = "protocols_statsd"
+harness = false
+
+[[bench]]
+name = "protocols_graphite"
+harness = false
```

Comment:
New line?

Reply:
Those tend to get auto-trimmed by my editor. Does TOML need a newline at the end that I'm not aware of?